src/containers/Ledgers/LedgerMetrics.tsx

```diff
@@ -0,0 +1,120 @@
+import { useTranslation } from 'react-i18next'
+import { Tooltip, useTooltip } from '../shared/components/Tooltip'
+import { renderXRP } from '../shared/utils'
+import PauseIcon from '../shared/images/ic_pause.svg'
+import ResumeIcon from '../shared/images/ic_play.svg'
+import './css/ledgerMetrics.scss'
+import { useIsOnline } from '../shared/SocketContext'
+import { useLanguage } from '../shared/hooks'
+
+const DEFAULTS = {
+  load_fee: '--',
+  txn_sec: '--',
+  txn_ledger: '--',
+  ledger_interval: '--',
+  avg_fee: '--',
+  quorum: '--',
+  nUnl: [],
+}
+
+export const LedgerMetrics = ({
+  data: suppliedData,
+  onPause,
+  paused,
+}: {
+  data: any
+  onPause: any
+  paused: boolean
+}) => {
+  const data = { ...DEFAULTS, ...suppliedData }
+  const { tooltip, showTooltip, hideTooltip } = useTooltip()
+  const { t } = useTranslation()
+  const isOnline = useIsOnline()
+  const language = useLanguage()
+
+  const renderPause = () => {
+    const Icon = paused ? ResumeIcon : PauseIcon
+    const text = paused ? 'resume' : 'pause'
+
+    return (
+      <div
+        tabIndex={0}
+        role="button"
+        className="pause-resume"
+        onClick={onPause}
+        onKeyUp={onPause}
+      >
+        <Icon className="icon" alt={t(text)} />
+        <span>{t(text)}</span>
+      </div>
+    )
+  }
+
+  if (data.load_fee === '--') {
+    data.load_fee = data.base_fee || '--'
+  }
+  delete data.base_fee
+  const items = Object.keys(data)
+    .map((key) => {
+      let content: any = null
+
+      let className = 'label'
+      if (data[key] === undefined && key !== 'nUnl') {
```

Comment:
Can you add a comment explaining why `nUnl` is special here? It's getting a lot of special treatment in this if/else - maybe there's a better way to separate out that logic? (ie. similar to how you deal with `base_fee` beforehand then delete it from the object?)